app/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnector.scala

```diff
@@ -81,6 +70,22 @@ class SubscriptionConnectorImpl @Inject()(http: HttpClient, val config: Services
     )
   }
 
+  override def getSubscription(cgtReference: CgtReference)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, HttpResponse] = {
+
+    val queryParameters = Map("regime" -> "CGT", "id" -> cgtReference.value)
+
+    EitherT[Future, Error, HttpResponse](
+      http
+        .get(subscriptionDisplayUrl, queryParameters, headers)
```

Comment:
this needs to be updated to point to the url in the DES spec: `/subscriptions/{regime}/{idType}/{idValue}`